Expensify/App
# Fix: App changes positions of Ui elements when phone language RTL

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7476

### Tests




### QA Steps


1. Set phone language to any RTL ( Settings  -> Languages & input -> Languages ) chose Arabic for example 
2. Open the app
3. Login & Browse
4. make sure the UI element is in its normal  position 
5. make sure the  normal text also does not align to right instead of left 
6. got to text input and use English keyboard (RTL language  not supported)
7. make sure text input aligns to left 
8. test on android and ios . and flow all previous steps. 



### Tested On


### Screenshots


#### Web

![Screen Shot 2022-02-07 at 2 22 50 AM](https://user-images.githubusercontent.com/21364901/152708094-adcf11c4-0d59-49a3-a8a3-fa8987a29876.png)

#### Mobile Web

![Screenshot_20220207-211827_Chrome.jpg](https://user-images.githubusercontent.com/21364901/152856683-83521792-81fd-484f-90de-f4d8667c13fa.jpg)

#### Desktop

![Screen Shot 2022-02-07 at 2 30 40 AM](https://user-images.githubusercontent.com/21364901/152708477-21fdccfc-2c65-4fbb-a6dc-18fb7d3f23ba.png)


#### iOS


![Screen Shot 2022-02-07 at 2 02 02 AM](https://user-images.githubusercontent.com/21364901/152707316-6795bb7d-7e94-4b97-b10c-1d527e64a5e1.png)
55b-d8534034caed.png)


https://user-images.githubusercontent.com/21364901/153756257-fa241dcb-c853-473f-bbe9-4d5c2bd2a2c5.mov


#### Android

![Screen Shot 2022-02-07 at 1 45 17 AM](https://user-images.githubusercontent.com/21364901/152706775-65553260-6ef4-4176-8e10-908e80a65524.png)
